### PR TITLE
LPI, CAPE and snowfall limit in ICON

### PR DESCRIPTION
**Pull request by patrick-zippenfenig**

### CODE REVIEW

**Review by drmrbrewer — COMMENTED** [review]
I guess that `ligthning_potential` should be corrected to `lightning_potential`

File: Resources/Views/docs-dwd-api.leaf [hunk 1]
```diff
@@ -202,6 +208,20 @@
...
+            name="hourly">
+          <label class="form-check-label" for="cape">
+            CAPE
+          </label>
+        </div>
+        <div class="form-check">
+          <input class="form-check-input" type="checkbox" value="ligthning_potential" id="ligthning_potential"
+            name="hourly">
+          <label class="form-check-label" for="ligthning_potential">
+            Ligthning Potential LPI <span class="text-muted"><sup>(2)</sup></span>
```
Comment (drmrbrewer) [inline Resources/Views/docs-dwd-api.leaf:222]: `Ligthning Potential LPI <span class="text-muted"><sup>(2)</sup></span>`

-->

`Lightning Potential Index LPI <span class="text-muted"><sup>(2)</sup></span>`

?

**Review by drmrbrewer — COMMENTED** [review]
Could LPI also be added to the general API, with a note about geographical availability (and for CAPE)?

File: Resources/Views/docs-dwd-api.leaf (hunk 1 above)
Comment (patrick-zippenfenig) [reply Resources/Views/docs-dwd-api.leaf]: Thanks, fixed

**Comment by patrick-zippenfenig** [comment]
> Could LPI also be added to the general API, with a note about geographical availability (and for CAPE)?

I have added it to the API variables, but I will not yet add it to the interface/documentation. It might be possible to calculate it for other domains like ICON-eu, GFS HRRR or GEM HRDPS. With better availability, I can add it to the generic weather API

File: Resources/Views/docs-dwd-api.leaf [hunk 2]
```diff
@@ -216,10 +216,10 @@
           </label>
         </div>
         <div class="form-check">
-          <input class="form-check-input" type="checkbox" value="ligthning_potential" id="ligthning_potential"
+          <input class="form-check-input" type="checkbox" value="lightning_potential" id="lightning_potential"
             name="hourly">
-          <label class="form-check-label" for="ligthning_potential">
-            Ligthning Potential LPI <span class="text-muted"><sup>(2)</sup></span>
+          <label class="form-check-label" for="lightning_potential">
+            Ligthning Potential Index LPI <span class="text-muted"><sup>(2)</sup></span>
```
[the remark this answers is not in the document]
Comment (patrick-zippenfenig) [reply Resources/Views/docs-dwd-api.leaf:222]: Thanks, fixed on main branch now

**Pull request merged**